locale/en.json

```diff
@@ -39,6 +39,7 @@
     "copy": "Copy"
   },
   "sidebar": {
+    "nodeFilter": "Node filter",
```

Comment:
I believe in Hopglass they call ist "hostname filter", which in my opinion describes it as accurate as possible.
Nodes can be filtered in a number of ways, so just calling it "node filter" might be confusing. As it is more than just a search, but a filter, I would stick to that terminology.